src/main/java/com/epam/ms/user/UserProfileHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.ms.user;
+
+import com.epam.ms.user.dto.UserProfile;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserProfileHandler {
+    private static final double WEIGHT_FACTOR = 9.99;
```

Comment:
also ,really convenient to have a property file for that
in spring boot they try to put EVERYTHING to config file. Then you can either use `@Value()` or! [`@ConfigurationProperties`](https://www.baeldung.com/configuration-properties-in-spring-boot) to inject a lot of configs at once.

Reply:
Thank you, done. Please review again, did I get the idea right?